### PR TITLE
11th newsletter, initial commit

### LINKED ISSUE #6: Newsletter 11
Reporting issue for the next newsletter, roughly scheduled for `2018-09-09`.

### PR DESCRIPTION
**Pull request by jamesmunns**
Closes #6 

I can update the social media links after the initial merge.

### CODE REVIEW

File: content/2018-09-09-newsletter-11.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+[fourth chapter]: https://twitter.com/branan/status/1038222375790161920
+[Exploring Rust on Teensy]: https://branan.github.io/teensy/
+[Branan Riley]: https://github.com/branan
+
+[Visual Inertial PCB]: https://twitter.com/korken89/status/1038494310830952449
+[Emil Fresk]: https://github.com/korken89
+
+[four-channel synthesizer]: https://twitter.com/therealjpster/status/1036298070022086656
+[Jonathan Pallant]: https://github.com/thejpster
+[demo audio clip]: https://thejpster.org.uk/beat.mp3
```
Comment (thejpster) [inline content/2018-09-09-newsletter-11.md]: Please could we swap thejpster.org.uk/beat.mp3 for https://www.youtube.com/watch?v=NvfyFhFK11g. Thanks.

Comment (jamesmunns) [reply content/2018-09-09-newsletter-11.md]: :+1: done!

**Comment by jamesmunns** [comment]
Can I self-bors?

bors r+

File: content/2018-09-09-newsletter-11.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+<!-- more -->
+
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-embedded/blog/issues/12
+
+## Highlights
+
+* `rustc` now supports 4 **Cortex-R** targets thanks to the work of [paoloteti] in the Cortex-R space! You can build programs for these targets using nothing but the Rust toolchain
+* [James Munns] gave a talk at RustConf 2018, covering the basics of embedded systems, and how Rust's Zero Cost Abstractions are a perfect match for bare metal systems
```
Comment (adamgreen) [inline content/2018-09-09-newsletter-11.md]: Should "at RustConf 2018" on this line be surrounded with square brackets?

Comment (jamesmunns) [reply content/2018-09-09-newsletter-11.md]: Yup, will fix in the next PR. Good catch!

**Pull request merged**